src/main/java/frc/robot/subsystems/ControlPanel.java

```diff
@@ -106,7 +106,7 @@ public ColorStates getTargetColor() {
     public CommandBase spinControlPanel(DoubleSupplier speed) {
         CommandBase cmd = new FunctionalCommand(() -> {
         }, () -> {
-            spinnerMotor.set(speed.getAsDouble());
+            spinnerMotor.set(-Math.abs(speed.getAsDouble()));
```

Comment:
Could we be using the speed controllers built in invert functionality instead of having negatives everywhere?

Reply:
yeah, we do need to look at why control panel is always trying to spin...